README.md

```diff
@@ -1,336 +1,243 @@
-   APIs - e.g. `XPostgreSQLInstance`.
-* `Compositions` offer the classes-of-service supported for each self-service
-   API - e.g. `Standard`, `Performance`, `Replicated`.
-
-![Upbound Overview](docs/media/compose.png)
-
-Crossplane `Providers` include the cloud service primitives (AWS, Azure, GCP,
-Alibaba) used in a `Composition`.
-
-Learn more about `Composition` in the [Crossplane
+## Overview
+
+This reference platform defines a custom API for creating an EKS cluster
+([XCluster](package/cluster/definition.yaml)) which includes the actual EKS
+cluster, a network fabric and Prometheus and other cluster services
+([XServices](package/cluster/composition.yaml)). Additionally it defines a
+custom API for provisioning Postgres Databases
+([XPostgreSQLInstance](package/database/postgres/definition.yaml)). 
+
+
+```mermaid
+graph LR;
+    MyApp(My App)---MyCluster(XRC: my-cluster);
+    MyCluster---XRD1(XRD: XCluster);
+    MyApp---MyDB(XRC: my-db);
+    MyDB---XRD2(XRD: XPostgreSQLInstance);
+		subgraph Configuration:upbound/platform-ref-aws;
+	    XRD1---Composition(XEKS, XNetwork, XServices);
+	    XRD2---Composition2(Composition);
+		end
+		subgraph Provider:upbound/provider-aws
+	    Composition---IAM.MRs(MRs: IAM Role, RolePolicyAttachment,OpenIDConnectProvider);
+	    Composition---EKS.MRs(MRs: EKS Cluster, ClusterAuth, NodeGroup);
+	    Composition2---RDS.MRs(MRs: RDS SubnetGroup, Instance);
+		end
+
+style MyApp color:#000,fill:#e6e6e6,stroke:#000,stroke-width:2px
+style MyCluster color:#000,fill:#D68A82,stroke:#000,stroke-width:2px
+style MyDB color:#000,fill:#D68A82,stroke:#000,stroke-width:2px
+style Configuration:upbound/platform-ref-aws fill:#f1d16d,opacity:0.3
+style Provider:upbound/provider-aws fill:#81CABB,opacity:0.3
+style XRD1 color:#000,fill:#f1d16d,stroke:#000,stroke-width:2px,stroke-dasharray: 5 5
+style XRD2 color:#000,fill:#f1d16d,stroke:#000,stroke-width:2px,stroke-dasharray: 5 5
+style Composition color:#000,fill:#f1d16d,stroke:#000,stroke-width:2px
+style Composition2 color:#000,fill:#f1d16d,stroke:#000,stroke-width:2px
+
+style IAM.MRs color:#000,fill:#81CABB,stroke:#000,stroke-width:2px
+style EKS.MRs color:#000,fill:#81CABB,stroke:#000,stroke-width:2px
+style RDS.MRs color:#000,fill:#81CABB,stroke:#000,stroke-width:2px
+```
+
+Learn more about Composite Resources in the [Crossplane
 Docs](https://crossplane.io/docs/v1.9/concepts/composition.html).
 
-## Pre-Requisite & Optional Tools
-
-Install the following command line tools:
-
-* `up cli`
-
-  There are multiple ways to [install up](https://cloud.upbound.io/docs/cli/#install-script), including Homebrew and Linux packages.
-
-  ```console
-  curl -sL https://cli.upbound.io | sh
+## Quickstart
 
-  ```
+### Pre-Requisites
 
-## Platform Ops/SRE: Run your own internal cloud platform
+Before we can install in the reference platform we need to install the `up` CLI
+and UXP. To install `up` run this install script:
```

Comment:
It's not clear to the reader what the UXP is.

Reply:
Addressed with https://github.com/upbound/platform-ref-aws/pull/79/commits/2173f8f7e649d251f62ad43dc57eeca07cdf199e